routers/web/repo/view.go

```diff
@@ -234,14 +234,11 @@ func getFileReader(ctx gocontext.Context, repoID int64, blob *git.Blob) ([]byte,
 	}
 
 	meta, err := git_model.GetLFSMetaObjectByOid(ctx, repoID, pointer.Oid)
-	if err != nil && err != git_model.ErrLFSObjectNotExist { // fallback to plain file
+	if err != nil { // fallback to plain file
```

Comment:
It's better to have an error or warning log here. Otherwise LGTM.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/31812/commits/b2543a6554944c9908e978718739009c67c29ef5